webpack.common.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+
+const isDev = process.env.NODE_ENV === 'development';
```

Comment:
hm, I think that `common` config should know nothing about whether it's dev or prod and should contain only config that is common for both. Does it make sense?